src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {
+  Route,
+  Routes,
+} from 'react-router-dom';
+import Books from './pages/Books';
+import Categories from './pages/Categories';
+import NavBar from './components/navbar';
+import AddBook from './components/form';
+
+function App() {
+  return (
+    <section>
+      <NavBar />
+      <Routes>
+        <Route exact path="/" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
+        <Route path="/books" element={<Books />} />
```

Comment:
- [ ] I am not sure why you are adding the `/books` route the `/` route is okay but if you wanted to use a wild card path to route the user to the `book` page I'd suggest using the `*` wildcard.

Reply:
This issue is well resolved! ✔️